scripts/globals/job_utils/warrior.lua

```diff
@@ -59,7 +67,15 @@ xi.job_utils.warrior.useBrazenRush = function(player, target, ability)
 end
 
 xi.job_utils.warrior.useDefender = function(player, target, ability)
-    player:addStatusEffect(xi.effect.DEFENDER, 1, 0, 180 + player:getMod(xi.mod.DEFENDER_DURATION))
+    local power    = 25
+    local duration = 180 + player:getMod(xi.mod.DEFENDER_DURATION)
+
+    if player:getMainJob() == xi.job.WAR then
+        local powerScale = math.max(0, (math.floor((utils.getActiveJobLevel(player, xi.job.WAR) - 40) / 10)) * 2)
+        power            = math.min(25 + powerScale, 35)
+    end
+
+    player:addStatusEffect(xi.effect.DEFENDER, power, 0, duration)
```

Comment:
local power        = 25
local duration     = 180 + player:getMod(xi.mod.DEFENDER_DURATION)
local warriorLevel = utils.getActiveJobLevel(player, xi.job.WAR)

power = power + math.floor((warriorLevel - 40) / 5)
power = utils.clamp(power , 25, 35)

player:addStatusEffect(xi.effect.DEFENDER, power, 0, duration)

Reply:
A simple matter of
```lua
local warriorLevel = player:getMainJob() == xi.job.WAR and player:getMainLevel() or 0
```
if so